Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -784,10 +803,10 @@ function logInfo {
 # In the past we copied to the users temp folder, difficult to find on machines with lots of profiles. Solved this by always placing the log in the normal Huntress folder.
 function copyLogAndExit {
     Start-Sleep 1
-    $logLocation = $DebugLog
-    $agentPath   = getAgentPath
-    Copy-Item -Path $logLocation -Destination $agentPath -Force
-    Write-Output "$($Debuglog) copied to $(getAgentPath)"
+    $agentPath = getAgentPath
+    $logLocation = Join-Path $agentPath "HuntressPoShInstaller.log"
+    Copy-Item -Path $DebugLog -Destination $logLocation -Force
+    Write-Output "$($DebugLog) copied to $(getAgentPath)"
```

Comment:
This could be `Write-Output "$($DebugLog) copied to $($agentPath)"` to prevent running the getAgentPath function again (probably doesn't impact performance, but the variable is set earlier anwyays